### PR TITLE
Unit NO:ORGNR population

### LINKED ISSUE #382: Unit NO:ORGNR population
orgnr

Organization number and VAT number in right upper corner should not be editable. Those values should be picked from `Fakturabank Firma ID` section:
NO:ORGNR -> Organization number
NO:VAT -> VAT number.

If those values are not selected, leave Organization number and VAT number empty, but if we click on that input, highlight `Fakturabank Firma ID` section where those values should be entered.

⚠️  Have in mind that when new supplier is created, it is possible to enter NO:ORGNR at once and Organization number is populated automatically. Investigate how it is possible to create `Fakturabank Firma ID` instead.

Issue reported by Arnt Telhaug

### PR DESCRIPTION
**Pull request by Baxxx**
Closes #382

### CODE REVIEW

File: modules/accountplan/view/reskontro.php [hunk 1]
```diff
@@ -135,15 +151,15 @@
     <td></td>
     <td><input class="lodoreqfelt" type="text" name="accountplan.AccountName" value="<? print $account->AccountName  ?>" size="30"><? if($account->AccountName) { ?><a href="http://w2.brreg.no/enhet/sok/treffliste.jsp?navn=<? print urlencode($account->AccountName) ?>" target="top">brreg</a><? } ?></td>
     <td style="text-align:right" colspan="2">Organisasjonsnummer (opplysninger hentes automatisk basert p&aring; orgnummer)</td>
-    <td><input class="lodoreqfelt" type="text" name="accountplan.OrgNumber" value="<? print $account->OrgNumber  ?>" size="30"><? if($account->OrgNumber) { ?><a href="http://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=<? print $account->OrgNumber ?>" target="top">brreg</a><? } ?></td>
+    <td><input class="lodoreqfelt read-only-input" type="text" name="accountplan.OrgNumber" value="<? print $account->OrgNumber  ?>" size="30" readonly="readonly" onclick="this.blur(); hightlightFirmaID();"><? if($account->OrgNumber) { ?><a href="http://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=<? print $account->OrgNumber ?>" target="top">brreg</a><? } ?></td>
```
Comment (Baxxx) [inline modules/accountplan/view/reskontro.php]: Instead of blur, focus the NO:ORGNR FirmaID field

File: modules/accountplan/view/record.inc [hunk 2]
```diff
@@ -83,18 +83,45 @@ function update_orgnr_vat_from_firmaid() {
...
+              LEFT JOIN accountplan ap ON aps.AccountPlanID = ap.AccountPlanID
+              WHERE ap.AccountPlanID is null;";
+    $_lib['db']->db_query($query);
+}
+
+function create_new_noorgnr_and_vat($AccountPlanID, $OrgNumber = '') {
+    global $_lib;
+    $query = "INSERT INTO accountplanscheme (AccountPlanID, FakturabankSchemeID, SchemeValue, CountryCode) VALUES
+              ('". $AccountPlanID ."', (SELECT FakturabankSchemeID FROM fakturabankscheme WHERE SchemeType = 'NO:ORGNR'), '". $OrgNumber ."', 'NO'),
+              ('". $AccountPlanID ."', (SELECT FakturabankSchemeID FROM fakturabankscheme WHERE SchemeType = 'NO:VAT'), '". $OrgNumber ."', 'NO');";
```
Comment (Baxxx) [inline modules/accountplan/view/record.inc]: Remove OrgNumber

File: modules/accountplan/view/record.inc [hunk 3]
```diff
@@ -83,18 +83,45 @@ function update_orgnr_vat_from_firmaid() {
     $_lib['db']->db_query("UPDATE accountplan SET VatNumber = '". $vatnr ."' WHERE AccountPlanID = '". $_POST["accountplan_AccountPlanID"] ."';");
 }
 
+function remove_hanging_firmaids() {
```
Comment (Baxxx) [inline modules/accountplan/view/record.inc]: Add comment what is this for

File: modules/accountplan/view/record.inc [hunk 4]
```diff
@@ -83,18 +83,45 @@ function update_orgnr_vat_from_firmaid() {
...
+    }
+}
+
+if($AccountPlanID) {
     // Check of this is a new and usaved account, if so - just create it
     $queryexist   = "select * from accountplan where AccountPlanID = $AccountPlanID";
     $accountexist = $_lib['storage']->get_row(array('query' => $queryexist));
+}
+
+if($_lib['input']->getProperty('action_accountplan_update') || $_lib['input']->getProperty('action_accountplan_updateautomatic') || $_lib['input']->getProperty('action_add_scheme') || ($_lib['input']->getProperty('action_save_scheme')) && !$accountexist)
```
Comment (Baxxx) [inline modules/accountplan/view/record.inc]: Fix the bracket

**Review by sls-lnluksa — APPROVED** [review]

**Review by stoivo — APPROVED** [review]

**Pull request merged**